pontoon/base/forms.py

```diff
@@ -159,15 +159,29 @@ def save(self, commit=True):
 )
 
 
-class UserFirstNameForm(forms.ModelForm):
+class UserProfileForm(forms.ModelForm):
     """
     Form is responsible for saving user's name.
     """
     first_name = forms.RegexField(regex='^[^<>"\'&]+$', max_length=30, strip=True)
+    email = forms.EmailField()
 
     class Meta:
         model = User
-        fields = ('first_name',)
+        fields = ('first_name', 'email')
+
+    def clean_email(self):
+        email = self.cleaned_data.get('email')
+        if(
+          email and
```

Comment:
This is the wrong level of indentation, you need to add 2 more spaces here and on all other lines of this block. 

Note that you can catch all those errors by running pylama locally. If you're using docker to run Pontoon, here's how you can do it: 

```bash
$ make dockershell
...
$ pylama pontoon
```